planner/core/common_plans.go

```diff
@@ -1116,6 +1120,38 @@ func (e *Explain) explainPlanInRowFormatCTE() (err error) {
 	return
 }
 
+func (e *Explain) explainPlanInRowFormatScalarSubquery() (err error) {
+	if e.TargetPlan == nil || e.TargetPlan.SCtx() == nil {
+		return nil
+	}
+	subqueries := e.TargetPlan.SCtx().GetSessionVars().StmtCtx.ScalarSubqueries
+	format := strings.ToLower(e.Format)
+	for i := range subqueries {
+		subquery := subqueries[i].(*ScalarSubquery)
+		id := fmt.Sprintf("ScalarSubquery_%d", i)
+		operatorInfo := fmt.Sprintf("output:%s", subquery.expr.ExplainInfo())
```

Comment:
This operator info is unnecessary?

Reply:
It shows the result of the subquery (in Explain Analyze) or the expression that references this subquery (in Explain).